### PR TITLE
Demquery

### PR DESCRIPTION
**Pull request by kylebarron**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] (Not applicable) If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/demquery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: demquery
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/28/dc/119312bc384c4a6526fdc0e4d655552516d812147c890539f92ad8924445/demquery-{{ version }}.tar.gz
+  sha256: 19650a37a5da271af5697d429db8f6cf6d43804dc46113fa622b80acd982d1f8
+  md5: e9792a03509ae3a3eec852f842e90090
```
Comment (marcelotrevisani) [inline recipes/demquery/meta.yaml]: Just one check is enough, you can delete the md5
```suggestion
```

File: recipes/demquery/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: demquery
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/28/dc/119312bc384c4a6526fdc0e4d655552516d812147c890539f92ad8924445/demquery-{{ version }}.tar.gz
```
Comment (marcelotrevisani) [inline recipes/demquery/meta.yaml]: Could you please change it to:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/demquery/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  md5: e9792a03509ae3a3eec852f842e90090
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/demquery/meta.yaml]: You will need to limit ``python`` here as well, please add
```suggestion
    - python >=3.5.0
```

**Comment by kylebarron** [comment]
@marcelotrevisani Thanks for the suggestions!

File: recipes/demquery/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5.0
+    - pip
```
Comment (marcelotrevisani) [inline recipes/demquery/meta.yaml]: you will need to add ``twine`` here
```suggestion
    - pip
    - twine
```

**Comment by marcelotrevisani** [comment]
I think it will not solve because you are not shipping the ``requirements.txt`` with the package and your ``setup.py`` needs it

**Comment by kylebarron** [comment]
Huh, you're right, it isn't in there.

File: recipes/demquery/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.1" %}
+{% set name = "demquery" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 19650a37a5da271af5697d429db8f6cf6d43804dc46113fa622b80acd982d1f8
```
Comment (marcelotrevisani) [inline recipes/demquery/meta.yaml]:
```suggestion
  sha256: 91c9ccaf8726c3b435e135151916dd1d87678e5ab8409c717fdf6fbece273914
```

**Comment by kylebarron** [comment]
Wow, checks actually passed! Thanks @marcelotrevisani! The errors seemed a little opaque.

**Pull request merged**